src/luxonis_ml/scheme/v1/config_building_blocks/base_models/metadata.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+from .custom_base_model import CustomBaseModel
+from ..enums import *
+
+class Metadata(BaseModel):
+    name: str
+    backbone: str
```

Comment:
Could also be omitted I think or is there a specific reason for keeping it?

Reply:
removing the backbone field you mean?

Reply:
yes